src/app/debug/filter-side-drawer/filter-side-drawer.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FilterService } from './filter.service';
+import { Subscription } from 'rxjs';
+import { animate, style, transition, trigger } from '@angular/animations';
+
+@Component({
```

Comment:
Please make this component standalone, have a look at the [docs](https://angular.io/guide/standalone-components). Don't forget to remove the declaration of the component in the app.module.ts

Reply:
FilterSideDrawer is now standalone